core/src/main/java/com/csi4999/screens/GenerationScreen.java

```diff
@@ -0,0 +1,234 @@
+    private Skin skin;
+    private final OrthographicCamera menuCam;
+    private final FitViewport menuViewport;
+    private final ALifeApp app;
+    private Stage stage;
+
+    private EnvProperties properties;
+
+    public GenerationScreen(ALifeApp app) {
+        this.app = app;
+        this.properties = EnvProperties.makeTestDefault();
+
+        skin = CustomAssetManager.getInstance().manager.get(SKIN_MAIN);
+
+        menuCam = new OrthographicCamera();
+        menuViewport = new FitViewport(400, 400, menuCam);
+
+        menuCam.position.set(menuCam.viewportWidth/2, menuCam.viewportHeight/2, 0);
+        menuCam.update();
+
+        stage = new Stage(menuViewport, app.batch);
+        Gdx.input.setInputProcessor(stage);
+    }
+
+    @Override
+    public void show() {
+        Table mainTable = new Table();
+
+        mainTable.setFillParent(true);
+        mainTable.top();
+
+        mainTable.align(Align.center);
+
+
+        TextButton continueButton = new TextButton("Continue", skin);
+        TextButton exitButton = new TextButton("Exit", skin);
+        exitButton.setColor(1f, 0f, 0f, 1f);
+
+        Slider mutationRateSlider = new Slider(0, 4, 0.1f, false, skin);
+        mutationRateSlider.setValue(properties.globalMutationRate);
+        Label mutationRateLabel = new Label("Mutation Rate", skin);
+        Label mutationRateValueLabel = new Label(String.valueOf(properties.globalMutationRate), skin);
+
+        Slider initialCreaturesSlider = new Slider(0, 1000, 1, false, skin);
+        initialCreaturesSlider.setValue(properties.initialCreatures);
+        Label initialCreaturesLabel = new Label("Initial Population", skin);
+        Label initialCreaturesValueLabel = new Label(String.valueOf(properties.initialCreatures), skin);
+
+        Slider initialFoodSlider = new Slider(0, 10000, 1, false, skin);
+        initialFoodSlider.setValue(properties.initialFood);
+        Label initialFoodLabel = new Label("Initial Food", skin);
+        Label initialFoodValueLabel = new Label(String.valueOf(properties.initialFood), skin);
+
+        Slider foodDeviationSlider = new Slider(0, 8192, 1, false, skin);
+        foodDeviationSlider.setValue(properties.foodSpawnStd);
+        Label foodDeviationLabel = new Label("Food Deviation", skin);
+        Label foodDeviationValueLabel = new Label(String.valueOf(properties.foodSpawnStd), skin);
+
+        Slider sensorSlider = new Slider(0, 24, 1, false, skin);
+        sensorSlider.setValue(properties.maxSensors);
+        Label sensorLabel = new Label("Max Sensors", skin);
+        Label sensorValueLabel = new Label(String.valueOf(properties.maxSensors), skin);
+
+        Slider toolSlider = new Slider(0, 24, 1, false, skin);
+        toolSlider.setValue(properties.maxTools);
+        Label toolLabel = new Label("Max Tools", skin);
+        Label toolValueLabel = new Label(String.valueOf(properties.maxTools), skin);
+
+
+        // Select which components can be used here too?
```

Comment:
yeah, maybe for starters we have checkboxes for enabling/disabling tools/sensors. also this screen is gonna get a lot larger so you should add multiple columns